mypy/build.py

```diff
@@ -746,7 +751,9 @@ def process(self) -> None:
                 sem_anal.modules[p].names[c[-1]] = SymbolTableNode(
                     MODULE_REF, tree, p)
 
-        if self.id != 'builtins':
+        if self.id == 'builtins':
+            tree.defs.extend(parse.implicit_builtins)
```

Comment:
There's probably some place where 'None' is created from whole cloth. If you can find that place, it might be easy enough to add True, False, `__debug__` there.